fedimint-cli/src/lib.rs

```diff
@@ -568,6 +570,10 @@ Examples:
         #[arg(long, default_value = "10")]
         limit: u64,
     },
+
```

Comment:
[nit] It may be helpful to add documentation

Reply:
Fixed